x/interchainquery/keeper/abci.go

```diff
@@ -11,10 +11,6 @@ import (
 	"github.com/Stride-Labs/stride/x/interchainquery/types"
 )
 
-const (
-	RetryInterval = 25
```

Comment:
Is this just an unused variable? If so, good to merge

Reply:
yes